modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-suma.adoc

```diff
@@ -140,37 +146,91 @@ Alternatively you may install [systemitem]``mgrpxy-zsh-completion`` or [systemit
 
 . Reboot the system.
 
-== Deploy {productname} Proxy on {sles} {bci-suma}
+[IMPORTANT]
+====
+{sle-micro} is designed to update itself automatically by default and will reboot after applying updates.
+However, this behavior is not desirable for the {productname} environment.
+To prevent automatic updates on your server, {productname} disables the transactional-update timer during the bootstrap process.
+
+If you prefer the {sle-micro} default behavior, enable the timer by running the following command:
+
+[source, shell]
+----
+systemctl enable --now transactional-update.timer
+----
+====
+
+To continue with deployment, see xref:installation-and-upgrade:container-deployment/suma/proxy-deployment-suma.adoc#deploy-suma-proxy-persistent-storage[].
+
+
+
+[[deploy-suma-proxy-sles]]
+=== Deploy on {sles} {bci-suma}
 
 Alternatively, you can install {productname} Proxy on {sles} {bci-suma}.
 The following procedure describes the main steps of the installation process.
 
 .Procedure: Installing {productname} Proxy on {sles} {bci-suma}
 . Locate and download {sles} {bci-suma} [literal]``.iso`` at https://www.suse.com/download/sles/.
+// FIXME Do we need the following step?
++
 . Make sure that the {scc} code you plan to use has the Extensions enabled.
 . Start the installation of {sles} {bci-suma}.
   .. On the [literal]``Language, keyboard and product selection`` select the product to install.
   .. On the [literal]``License agreement`` read the agreement and check [guimenu]``I Agree to the License Terms``.
 . Select the registration method.
+
++
+
+[[NOTE]]
+====
+Please note that for {sles} {bci-suma}, you are required to have a valid {sles} subscription and corresponding regcode, which you must provide on this screen.
+You will be required to enter the {productname} Extension regcode below.
+====
 . In the screen [literal]``Extensions and Modules Selection`` check the following:
-  * {productname} Proxy Extension
++
+
+* {productname} Proxy Extension
   * Basesystem Module
   * Containers Module
++
+
+Enter the {productname} Extension regcode.
 . Complete the installation.
 . When the installation completes, log in to the newly installed server as root.
-. As root, run
+. Update the System (optional, if the system was not set to
+download updates during install):
 
 +
 
+[source,shell]
+----
+zypper up
+----
+
++
+
+// FIXME: is "Reboot" required here?  Cf. server counterpart
+
+. As root install podman and additional tools:
```

Comment:
```suggestion
. Log in as root and install [package]``podman`` plus [package]``mgrpxy`` and [package]``mgrpxy-bash-completion`` (if not already automatically installed):
```
Suggestion as per what you did for the server docs with [`9417259` (#3830)](https://github.com/uyuni-project/uyuni-docs/pull/3830/commits/941725998e2e5c99f41d28e063248d64969442ea)